### PR TITLE
Remove application.md and add jinja-markdown

### PR DESCRIPTION
**Pull request by tsnow03**

### CODE REVIEW

**Comment by tsnow03** [comment]
How do I get the repo2docker action to trigger here? It won't go and I think it has to do with NO_PUSH being explicitly disabled, but I'm not sure.

**Comment by weiji14** [comment]
> How do I get the repo2docker action to trigger here? It won't go and I think it has to do with NO_PUSH being explicitly disabled, but I'm not sure.

The repo2docker action that pushes a docker image to quay.io is only triggered on pushes to the main branch according to https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/43acedc1577a37eb6c349b43b52e334e944e2003/.github/workflows/repo2docker.yaml#L1-L5 I.e. if you merge this Pull Request, it should trigger.

**Comment by weiji14** [comment]
Btw, I just added `cookiecutter` to the environment.yml file at #8, which should fix the 'NetlifyPreview' and 'Test' workflows. Try updating the branch and see if it works.

**Comment by tsnow03** [comment]
We are now missing jinja. I think these are the list we haven't yet included from the hackweek env.yml website/book section.

**Comment by weiji14** [comment]
> We are now missing jinja. I think these are the list we haven't yet included from the hackweek env.yml website/book section.

Oops, my bad, shouldn't have left it out from #2... I can add it tomorrow (getting a bit late for me here), or you can open up a PR to add it too.

**Comment by tsnow03** [comment]
I'll try to reconcile this in the environment.

**Comment by sgibson91** [comment]
@tsnow03 you can get r2d to build and push on PRs by removing this line:

https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/0f11ed8091dba30654350c80fcb88136bd250469/.github/workflows/repo2docker-PR.yaml#L23

File: conda/environment.yml [hunk 1]
```diff
@@ -20,5 +20,10 @@ dependencies:
   - sphinxcontrib-bibtex
   - vim
   - wget
+  - jinja2 >= 2.11
+  - pygments>=2.6.1
+  - pymdown-extensions>=7.1
+  - pip:
+    - jinja-markdown==1.210911
```
[the remark this answers is not in the document]
Comment (tsnow03) [reply conda/environment.yml:27]: I'm having permission denied issues

[the remark this answers is not in the document]
Comment (tsnow03) [reply conda/environment.yml:27]: Fixed

File: conda/conda-lock.yml [hunk 2]
```diff
@@ -9,17 +9,18 @@
 # To update a single package to the latest version compatible with the version constraints in the source:
 #     conda-lock lock --lockfile conda-lock.yml --update PACKAGE
 # To re-solve the entire environment, e.g. after changing a version constraint in the source file:
-#     conda-lock -f /home/leong81/Documents/github/CryoCloudWebsite/conda/environment.yml --lockfile conda-lock.yml
+#     conda-lock -f /home/leong81/Documents/github/CryoCloudWebsite/conda/environment.yml -f /Volumes/GoogleDrive/.shortcut-targets-by-id/0B9H-XP5YRibEY1Brdk1qTzdxUm8/LandsatSST/Scripts/CryoCloudWebsite/conda/environment.yml --lockfile conda-lock.yml
```
Comment (weiji14) [inline conda/conda-lock.yml:12]: Not sure how this got here, but might want to remove this

```suggestion
#     conda-lock -f /home/leong81/Documents/github/CryoCloudWebsite/conda/environment.yml --lockfile conda-lock.yml
```

File: book/_config.yml [hunk 3]
```diff
@@ -15,9 +15,9 @@ parse:
     github_org_url: "https://github.com/CryoInTheCloud"
     book_repo: "CryoCloudWebsite"
     website_url: "https://cryointhecloud.github.io"
-    jupyterhub_url: "https://cryo.2i2c.cloud"
+    jupyterhub_url: "https://cryonthecloud.2i2c.cloud"
     slack_workspace_url: "https://CryoCloud.slack.com"
-    docker_image: "https://hub.docker.com/r/cryocloud"
+    docker_image: "https://hub.docker.com/r/cryointhecloud"
```
Comment (weiji14) [inline book/_config.yml:20]: Did you update the `DOCKER_USERNAME` and `DOCKER_PASSWORD` secrets for this new Docker Hub user account? Or are they under the same account?

Comment (tsnow03) [reply book/_config.yml:20]: The config wasn't correct and I don't think that line is used for anything in Github is it? The full link is: https://hub.docker.com/repository/docker/cryointhecloud/cryocloudwebsite. I think the config file is solely for the book and is filling in variables in certain md files. Our DOCKER_IMAGE secret is set to cryointhecloud/cryointhecloud and hasn't been changed.

File: conda/conda-lock.yml (hunk 2 above)
Comment (tsnow03) [reply conda/conda-lock.yml:12]: My computer made a weird reference to my local folder. My computer was having weird file and permission errors. But it should just be the source of the env.yml file that was used so your file system was added and then mine was.

**Comment by tsnow03** [comment]
I'm really not sure where this is coming from in the repo2docker action 
> denied: requested access to the resource is denied

File: conda/conda-lock.yml (hunk 2 above)
Comment (weiji14) [reply conda/conda-lock.yml]: Oops, just saw my username in there :sweat: Should use a generic path instead, but let's do it later...

File: book/_config.yml (hunk 3 above)
Comment (weiji14) [reply book/_config.yml:20]: Ah ok, I thought we were pushing to `cryocloud` before, but I see the docker image is at https://hub.docker.com/r/cryointhecloud/cryocloudwebsite/tags. Sorry for the confusion!

**Comment by weiji14** [comment]
> I'm really not sure where this is coming from in the repo2docker action
> 
> > denied: requested access to the resource is denied

Looking at https://github.com/CryoInTheCloud/CryoCloudWebsite/actions/workflows/repo2docker-PR.yaml, it seems like the workflow worked a while ago though. What change was made to make it different since then?

image

**Pull request merged**